src/test/resources/application.conf

```diff
@@ -0,0 +1,32 @@
+conversion-config {
```

Comment:
Either drop this one or the one in `src/main/resources/` 
Why? Because otherwise you have to maintain two different configs which is error prone. 
Additionally, it might make your tests pass, even if your code is broken. Why? Because, you forgot to adapt the config you run the tests on but you adapted the config for production. -> Always have centralized / the same configs for test and production if possible.